### DISCUSSION TITLE
Cannot connect to Oracle Db

### DISCUSSION THREAD

**Discussion by techlock77** [dbeaver/dbeaver#12494] [Q&A] [answered]
System information: 
- Operating system (distribution) and version -> WIN 10
- DBeaver version -> 21.0.3

Connection specification:
- Database name and version -> Oracle
- Driver name -> oracle.jdbc.OracleDriver
![image](https://user-images.githubusercontent.com/66756251/117880247-e5211380-b275-11eb-9050-6491ea4860e8.png)

**Comment by uslss** [comment]
Seems it is server-side issue. Please, check connection preferences
https://stackoverflow.com/questions/10786782/ora-12514-tnslistener-does-not-currently-know-of-service-requested-in-connect-d

**Comment by techlock77** [comment]
The problem is our connection uses service_name but on Dbeaver there is no slot where we can edit the service_name.

![image](https://user-images.githubusercontent.com/66756251/117986081-49d87e80-b307-11eb-92db-2dcaeac1e7a3.png)

**Comment by techlock77** [comment]
@uslss Is there a way to edit the service name in Dbeaver>?

**Comment by p91paul** [comment, marked as answer]
@techlock77 in your screenshot, you are connecting via service name (orcl). If you actually want to use SID instead of service name, change that combo to SID and orcl will be interpreted as SID instead of service name.
